src/Test/Integration/FeatureContextAbstract.php

```diff
@@ -60,6 +60,13 @@
     abstract protected function getStorageNames();
 
     /**
+     * @see initScenario()
```

Comment:
Just out of curiosity, why is this needed? Not that it's bad, I just don't see the point :) Isn't it possible to just override the initScenario like this?
```
public function initScenario()
{
  parent::initScenario();
  // do something
}```